bitnami/etcd/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: etcd
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/etcd
   - https://coreos.com/etcd/
-version: 8.10.1
+version: 8.10.2
```

Comment:
A [newer version](https://github.com/bitnami/charts/blob/main/bitnami/etcd/Chart.yaml#LL27C10-L27C15) has been released. At this moment it is on `8.11.2`, please bump it to `8.11.3`.